app/utils/createEntityReducer.js

```diff
@@ -99,7 +99,26 @@ export function entities(key: string, fetchType?: ?AsyncActionType) {
   };
 }
 
-export function optimistic(mutateType?: ?AsyncActionType) {
+/* This part of CER will handle deletion of values of a given entity.
+ * For actions of the type given into `types.delete` when setting up CER will be deleted.
+ *
+ * Make sure to set `meta.id` in the action.
+ */
+export function deleteEntities(key: string, deleteType: AsyncActionType) {
+  return (state: any = defaultState, action: any) => {
+    const resultId = action.meta && action.meta.id;
+
+    if (!resultId || action.type !== get(deleteType, 'SUCCESS')) return state;
```

Comment:
Do we need to use `get` when deleteType is not optional?

Reply:
Yeah, good catch. No need to have optional types here